README.md

```diff
@@ -23,6 +23,8 @@ The above step should take about 15 minutes depending on the speed of your machi
 $ vagrant provision
 ```
 
+(If you are updating an older checkout, `refinery/config/config.json` will not be regenerated automatically: Either clear it, or add the missing keys from the error message.)
+
```

Comment:
This info is [already available](https://github.com/refinery-platform/refinery-platform/wiki/Troubleshooting#improperly-configured-configjson)